Cargo.toml

```
@@ -14,19 +14,19 @@ tracing = "0.1.36"
 tracing-subscriber = "0.3.15"
 once_cell = "1.9.0"
 gettext-rs = { version = "0.7.0", features = ["gettext-system"] }
-gtk = { package = "gtk4", version = "0.5.4" }
-gdk-wayland = { package = "gdk4-wayland", version = "0.5.4" }
-gdk-x11 = { package = "gdk4-x11", version = "0.5.4" }
-adw = { package = "libadwaita", version = "0.2.1", features = ["v1_2"] }
-gst_video = { package = "gstreamer-video", version = "0.19.4" }
-gst_pbutils = { package = "gstreamer-pbutils", version = "0.19.2", features = [
+gtk = { package = "gtk4", version = "0.6.6", features = ["v4_10"] }
```

You could also use `"gnome_44"` as feature instead, so it will be automatic with other libs too

```suggestion
gtk = { package = "gtk4", version = "0.6.6", features = ["gnome_44"] }
```